cpp/include/cudf/io/text/data_chunk_source_factories.hpp

```diff
@@ -27,16 +27,51 @@ namespace cudf::io::text {
 
 /**
  * @brief Creates a data source capable of producing device-buffered views of the given string.
+ * @param data the host data to be exposed as a data chunk source. Its lifetime must be at least as
+ *             long as the lifetime of the returned data_chunk_source.
+ * @return the data chunk source for the provided host data. It copies data from the host to the
+ *         device.
  */
 std::unique_ptr<data_chunk_source> make_source(host_span<const char> data);
 
 /**
  * @brief Creates a data source capable of producing device-buffered views of the file
+ * @param data the filename of the file to be exposed as a data chunk source.
```

Comment:
I spent some time checking the Doxygen config file but didn't find anything suspicious since the header path is added in `INPUT`
https://github.com/rapidsai/cudf/blob/d6952ba42bd815e8a54e02b907c3790ce26ea488/cpp/doxygen/Doxyfile#L837
and `RECURSIVE` is `YES`
https://github.com/rapidsai/cudf/blob/d6952ba42bd815e8a54e02b907c3790ce26ea488/cpp/doxygen/Doxyfile#L882
`include/cudf/io/text` is not in the exclusion path:
https://github.com/rapidsai/cudf/blob/d6952ba42bd815e8a54e02b907c3790ce26ea488/cpp/doxygen/Doxyfile#L907-L909

Not sure what's going wrong here.

Reply:
I'll dig into this tomorrow :)

Reply:
It should throw an error here. I tested it locally too. It did not throw warning.
I checked the history of Doxyfile. it's modified, but nothing that would affect this. will debug further.

Reply:
@upsj @PointKernel Found the issue. Anthing inside double nested namespace is not throwing error right now.
Tried 1.8.20 and 1.9.1. Same issue persists.
We have not encountered this error until now because all double nested namespace are inside `detail` namespace, which are in omitted detail files.
So, only files with double nested namespaces will be missed by doxygen.
CI checks still work for other files.

Can't find a workaround yet.

Reply:
> Anthing inside double nested namespace is not throwing error right now.

Thanks for digging this up. Doxygen is tough to deal with... :disappointed: